### ISSUE TITLE
whats the format of the music ?

### ISSUE THREAD

**Issue by RetroGuyYT** [Ptrk25/eshopMusicChanger#2]
just that.

**Comment by flopinou** [comment]
It's not BCSTM like themes?

**Comment by Masamune3210** [comment]
People mention on forums extracting the files and then appending .wav to play them, and the script calls for aac, so I'm willing to bet the answer is no

**Comment by flopinou** [comment]
Yes I found the original music file in the data of the eshop and it is in aac!
